src/content/protocols/compound-v3-arbitrum.md

```diff
@@ -0,0 +1,215 @@
+
+The permissions for all Comet contracts (USDC, WETH, wsETH, USDT, USDS) are similar and therefore only
+represented once as `Comet Proxy` and `Comet Implementation`in the table above.
+
+## Dependencies
+
+The compound-v3 protocol relies on a Chainlink oracle feed to price collateral and base assets in the system. The protocol does not validate asset prices returned by Chainlink feeds other than checking for a zero-value. The protocol further does not offer a fallback pricing mechanism in case the Chainlink oracle feeds are stale or untrusted. If not performing as expected, Chainlink oracle feeds can only be replaced through a regular Compound governance proposal with a delay (see Exit Window).
+
+The Chainlink oracle system itself is upgradeable potentially resulting in the publishing of unintended or malicious prices. The permissions to upgrade are controlled by a [multisig account](https://etherscan.io/address/0x21f73D42Eb58Ba49dDB685dc29D3bF5c0f0373CA) with a 4-of-9 signers threshold. This multisig account is listed in the Chainlink docs but signers are not publicly announced. The Chainlink multisig thus does not suffice the Security Council requirements specified by either L2Beat or DeFiScan resulting in a High centralization score.
+
+## Upgrade process
+
+Any market parameter change requires a new `Comet` deployment. The process is as follows:
+
+1.  new parameters are set using setters in the `Configurator` contract.
+2.  the `ProxyAdmin` contract uses `deployAndUpgradeTo`:
+    1. Deploys a new comet contract through the `Configurator`, which uses the `CometFactory`.
+    2. Updates the corresponding `Comet Proxy` to point to this newly deployed contract.
+
+A malicious update could simply perform the update to the `Comet Proxy`to introduce a malicious `Comet Implementation` contract, effectively stealing funds.
+This is because the `upgrade` function can still be called by the DAO in addition to `deployAndUpgradeTo`, allowing the DAO to deploy `Comet` contracts not created by the `Configurator`.
+
+The process is illustrated below.
+
+![Update scheme for a comet contract](./diagrams/compound-v3-update.png)
+
+# Security Council
+
+A security council called `Pause Guardian` has the power to pause all deposits, withdrawals, and transfers
+in the Arbitrum `Comet` contracts. The guardian is currently a 4/7 multisig made of Compound DAO community members and other trusted entities. The signers are announced [here](https://www.comp.xyz/t/community-multisig-4-of-6-deployment/134/18).
```

Comment:
Signers here dont match with onchain data

Change wording of "The guardian is currently a 4/7 multisig made of Compound DAO community members and other trusted entities."

to something like "The guardian is currently a 4/7 multisig made of a diverse set of signers including more than 50% outsiders (community members, non-affiliated entities)."

Reply:
It does not match the last post but all addresses of this Puase guardian are announced in the thread with their owners. Should we still consider them "not publicly announced"?

Reply:
yes so here they announce the "latest signer set" but this doesnt match with the onchain data https://www.comp.xyz/t/community-multisig-4-of-6-deployment/134/19